Makefile

```diff
@@ -0,0 +1,15 @@
+init: migrate
+	go build main.go
+
+migrate:
```

Comment:
i think we can change this command to "migrate.up"

Reply:
Sure.

Reply:
```suggestion
migrate.up:
```